src/org/traccar/database/DeviceManager.java

```diff
@@ -316,7 +316,16 @@ public void removeGroup(long groupId) throws SQLException {
         groupsById.remove(groupId);
     }
 
-    public String lookupAttribute(long deviceId, String attributeName) {
+    public String lookupServerAttribute(long deviceId, String attributeName) {
```

Comment:
I think if we keep them in the same area of the DeviceManager class, it should be fine. Don't need a separate class for it.